DESCRIPTION

```diff
@@ -30,11 +32,14 @@ Imports:
     data.table,
     XML,
     xml2,
-    jsonlite
+    jsonlite,
+    RJSONIO,
+    httr
 Suggests:
     testthat,
-    knitr
-RoxygenNote: 6.1.1
+    knitr,
+    markdown
```

Comment:
did you mean to use the package `rmarkdown` here?

Reply:
This was to resolve an issue we had when running `devtools::check_win_devel()` (see comment here: https://github.com/ropensci/rsnps/issues/102#issuecomment-658233790).